RecoTauTag/RecoTau/test/rerunMVAIsolationOnMiniAOD_Phase2.cc

```diff
@@ -0,0 +1,191 @@
+#include "FWCore/Framework/interface/EDAnalyzer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "DataFormats/JetReco/interface/GenJet.h"
+#include "DataFormats/PatCandidates/interface/Tau.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+#include "CommonTools/UtilAlgos/interface/TFileService.h"
+
+#include "TH1D.h"
+#include "TH2D.h"
+#include "TGraphAsymmErrors.h"
+
+class rerunMVAIsolationOnMiniAOD_Phase2 : public edm::EDAnalyzer {
```

Comment:
legacy analyzers should not be used
this should be `edm::one::EDAnalyzer<edm::one::SharedResources>` since it uses `TFileService`